docs/deployment/visual-studio-properties.md

```diff
@@ -0,0 +1,666 @@
+
+- **Application Display Version**
+
+  The version of the application. This should be a single digit integer. Defaults to `1`.
+
+## Android
+
+These are Android-specific .NET MAUI settings.
+
+### Manifest
+
+Settings related to the Android manifest.
+
+- **Application name**
+
+  The string that's displayed as the name of the application. This is the name that's shown in the app's title bar. If not set, the label of the app's MainActivity is used as the application name. The default setting is `@string/app_name` refers to the string resource `app_name` location in `Resources/values/Strings.xaml`.
+
+- **Application package name**
+
+  A string that's used to uniquely identify the application. Typically, the package name is based on a reversed internet domain name convention, such as `com.company.appname`.
+
+- **Application icon**
+
+  Specifies the application icon resource that will be displayed for the app. The setting `@drawable/icon` refers to the image file `icon.png` located in the `Resources/mipmap` folder.
+
+- **Application theme**
+
+  Sets the UI style that's applied to the entire app. Every view in the app applies to the style attributes that are defined in the selected theme.
+
+- **Application version number**
+
+  An integer value greater than zero that defines the version number of the app. Higher numbers indicate more recent versions. This value is evaluated programmatically by Android and by other apps, it isn't shown to users.
+
+- **Application version name**
+
+  A string that specifies the version of the app to users. The version name can be a raw string or a reference to a string resource.
+
+- **Install location**
+
+  Indicates preference as to where the app should be stored, whether in internal or external storage. Internal-only:  Prefer external:
+
+  | Option | Description |
+  | -- | -- |
+  | `Internal-only` | (Default) Specifies that the app can't be installed or moved to external storage. |
+  | `Prefer external` | Specifies that the app should be installed in external storage, if possible. |
+  | `Prefer internal` | Specifies that the app should be installed in internal storage, if possible. |
+
+- **Minimum Android version**
+
+  The oldest API level of an Android device that can install and run the app. Also referred to as `minSdkVersion`.
+
+- **Target Android version**
+
+  The target API level of the Android device where the app expects to run. This API level is used at run-time, unlike Target Framework, which is used at build time. Android uses this version as a way to provide forward compatibility. Also referred to as `targetSdkVersion`, this should match Target Framework `compileSdkVersion`.
+
+### Options
+
+Miscellaneous options for building an Android app.
+
+- **Android package format**
+
+  Either `apk` or `bundle`, which packages the Android application as an APK file or Android App Bundle, respectively. This can be set for individually for both Debug and Release modes.
```

Comment:
"This can be set individually for...."?

Reply:
```suggestion
  Either `apk` or `bundle`, which packages the Android application as an APK file or Android App Bundle, respectively. This can be set individually for both Debug and Release modes.
```